apps/frontend/src/modules/auth/auth-api.ts

```diff
@@ -36,6 +38,28 @@ class AuthApi extends BaseHTTPApi {
 		return await response.json<UserDto>();
 	}
 
+	public async requestResetPassword(payload: EmailDto): Promise<null> {
+		await this.load(this.getFullEndpoint(AuthApiPath.FORGOT_PASSWORD, {}), {
+			contentType: ContentType.JSON,
+			hasAuth: false,
+			method: "POST",
+			payload: JSON.stringify(payload),
+		});
+
+		return null;
+	}
+
+	public async resetPassword(payload: ResetPasswordDto): Promise<null> {
+		await this.load(this.getFullEndpoint(AuthApiPath.RESET_PASSWORD, {}), {
+			contentType: ContentType.JSON,
+			hasAuth: false,
+			method: "PATCH",
+			payload: JSON.stringify(payload),
+		});
+
+		return null;
```

Comment:
Return a new user from backend and directly add to the store (the user will be logged in)

Reply:
They will just enter the same password they entered few seconds ago, I don't think it'll affect security